README.md

```diff
@@ -52,36 +53,17 @@ When a branch is merged into master, CI runs. If the CI passes, the main deploy
 
 Note that `docker-compose.yml` and `package.json` at the root level must be kept in sync.
 
-## Saving and restoring the database
```

Comment:
My impression is that we no longer do this stuff. If in fact we do, we should of course not delete it.